sycl/test/graph/graph-record-dotp.cpp

```diff
@@ -50,21 +48,24 @@ int main() {
     });
   });
 
-  q.submit([&](sycl::handler &h) {
+  auto eventA = q.submit([&](sycl::handler &h) {
+    h.depends_on(initEvent);
```

Comment:
Is this PR enough to satisfy https://github.com/reble/llvm/issues/89? Or do we still need to verify that `h.depends_on(nodeEvent);`  inside an explicit API added command-group works?